### PR TITLE
slight code cleanup

### PR DESCRIPTION
**Pull request by anmousyon**
cleaned up if else with date_format
made build_toolbars() more consistent and clear

### CODE REVIEW

File: reddit_liveupdate/utils.py [hunk 1]
```diff
@@ -10,30 +10,23 @@
 
 def pretty_time(dt, allow_relative=True):
     ago = datetime.datetime.now(pytz.UTC) - dt
+    date_format = ''
```
Comment (13steinj) [inline reddit_liveupdate/utils.py]: Just as a btw this line isn't needed since `date_format` is being set in all branches of the if statement

Comment (anmousyon) [inline reddit_liveupdate/utils.py]: Good note, fixed the line.

**Review by spladug — APPROVED** [review]
Thanks! Looks reasonable, I'll merge and deploy next week.

**Comment by spladug** [comment]
I've squashed this down and merged it into the `devel` branch which is where I'm staging stuff for deploy. Going to close this out now. Thanks again for the contribution! Do you have a reddit username that I can gift the open sorcerer trophy to?

**Pull request closed without merging**